### DISCUSSION TITLE
Hyperion Android Grabber Support?

### DISCUSSION THREAD

**Discussion by oerix** [awawa-dev/HyperHDR#824] [Q&A] [answered]
Hi, just wondering does HyperHDR work with with Hyperion Android Grabber in anyway? If so, how do I go about setting it up? I can't seem to get it working with HyperHDR only Hyperion.

Thanks

**Comment by awawa-dev** [comment, marked as answer]
Hi
Yes, it's working even with HDR tone mapping support. You must first enable protobuf service in the network tab.

**Reply by MotionJVE** [reply]
Is there something you need to do to get done mapping to work? Enabling/disabling it in settings doesn't do anything for me

**Comment by oerix** [comment]
> Hi Yes, it's working even with HDR tone mapping support. You must first enable protobuf service in the network tab.

Ah great, thank you! :)
